File: UI/wwwroot/Model/Tipo_notas.js
//@ts-check
// @ts-ignore
import { EntityClass } from '../WDevCore/WModules/EntityClass.js';
import { Calificaciones } from './Calificaciones.js';
class Tipo_notas extends EntityClass {
   /** @param {Partial<Tipo_notas>} [props] */
   constructor(props) {
       super(props, 'EntityDbo');
       for (const prop in props) {
           this[prop] = props[prop];
       }
   }
   /**@type {Number}*/ Id;
   /**@type {String}*/ Nombre;
   /**@type {String}*/ Nombre_corto;
   /**@type {Number}*/ Periodo_lectivo_id;
   /**@type {Number}*/ Consolidado_id;
   /**@type {Number}*/ Numero_consolidados;
   /**@type {String}*/ Observaciones;
   /**@type {Date}*/ Created_at;
   /**@type {Date}*/ Updated_at;
   /**@type {Number}*/ Orden;
   /**@type {Array<Calificaciones>} OneToMany*/ Calificaciones;
}
export { Tipo_notas };

File: CAPA_NEGOCIO/Gestion_Notas/Model/Tipo_notas.cs
using CAPA_DATOS;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
namespace DataBaseModel {
   public class Tipo_notas : EntityClass {
       [PrimaryKey(Identity = true)]
       public int? Id { get; set; }
       public string? Nombre { get; set; }
       public string? Nombre_corto { get; set; }
       public int? Periodo_lectivo_id { get; set; }
       public int? Consolidado_id { get; set; }
       public int? Numero_consolidados { get; set; }
       public string? Observaciones { get; set; }
       public DateTime? Created_at { get; set; }
       public DateTime? Updated_at { get; set; }
       public int? Orden { get; set; }
     
   }
}
